expression/builtin_time_vec_test.go

```diff
@@ -115,6 +115,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.DayOfMonth: {
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDatetime}},
 	},
+	ast.DayName: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETDatetime}},
```

Comment:
Please add more cases to test `vecEvalInt` and `vecEvalReal`.

Reply:
we can't, because `ETInt` and `ETReal` will convert to `ETString` in this line